### PR TITLE
feat: sample conversion rate (%) in a reaction scheme

### PR DESCRIPTION
**Pull request by adambasha0**
- [ ] rather 1-story 1-commit than sub-atomic commits

- [x] commit title is meaningful =>  git history search

- [x] commit description is helpful => helps the reviewer to understand the changes

- [ ] code is up-to-date with the latest developments of the target branch (rebased to it or whatever) => :fast_forward:-merge for linear history is favoured

- [x] added code is linted

- [x] tests are passing (at least locally): we still have some random test failure on CI. thinking of asking spec/examples.txt to be commited

- [ ] in case the changes are visible to the end-user,  video or screenshots should be added to the PR => helps  with user testing

- [ ] testing coverage improvement is improved.

- [ ] CHANGELOG :  add a bullet point on top (optional: reference to github issue/PR )

- [ ] parallele PR for documentation  on docusaurus  if the feature/fix is tagged for a release

### CODE REVIEW

**Comment by adambasha0** [comment]
[Chemotion dev (8).webm](https://github.com/user-attachments/assets/89dd74f3-c5b2-45a8-9fe6-539e1ac46674)

**Comment by adambasha0** [comment]
[Chemotion dev (9).webm](https://github.com/user-attachments/assets/057953a0-1b51-420c-8fea-500aef8e9a52)

**Review by TasnimMehzabin — APPROVED** [review]

File: app/packs/src/components/common/ToggleButton.js [hunk 1]
```diff
@@ -24,9 +28,9 @@ export default function ToggleButton({
         className={`toggle-button ${isToggled ? 'on' : 'off'}`}
         size="xs"
         onClick={handleChange}
-        style={{ backgroundColor: buttonColor, minWidth: '50px', border: 'none' }}
+        style={{ backgroundColor: buttonColor, border: 'none', padding: '0px 0px 0px 0px' }}
```
Comment (ashwiniHerle) [inline app/packs/src/components/common/ToggleButton.js:31]: This inline style can be replaced with corresponding bootstrap utility classes

Comment (adambasha0) [reply app/packs/src/components/common/ToggleButton.js]: done

**Pull request merged**